SERVER/CONTROLLERS/userAcc_Controler.js

```diff
@@ -1,71 +1,71 @@
-import accounts from '../MODELS/User_Accounts';
+import pool from '../test/MODELS/create';
 import create_acc_schema from '../JOI_VALIDATION/create_acc_validation';
 import login_schema from '../JOI_VALIDATION/login_user_validation';
 import Joi from '@hapi/joi';
 import jwt from 'jsonwebtoken';
 import dotenv from 'dotenv';
 
 class userAccountControler{
-    //Create account
     createUseraccount(req, res){
         dotenv.config();
         const user_secret= process.env.SECRET;
         const {firstName, lastName, email, password, address, bio, occupation, expertise}= req.body;
         const create_acc_validation = Joi.validate(req.body, create_acc_schema);
         if(create_acc_validation.error){
             const signup_errors=[];
-            for(let index=0; index<create_acc_validation.error.details.length; index++){
-                signup_errors.push(create_acc_validation.error.details[index].message.split('"').join(" "));
+            for(let i=0; i<create_acc_validation.error.details.length; i++){
+                signup_errors.push(create_acc_validation.error.details[i].message.split('"').join(" "));
             }
             return res.status(400).send({
                 status: 400,
                 error: signup_errors[0]
             });
         }else{
-            const all_user_accounts= accounts.AllAccounts;
-            const acc_exists= all_user_accounts.find(acc=>acc.email===email);
-            if(acc_exists){
-                return res.status(409).json({
-                    status:409,
-                    message: `This account already exists.Try another one.`
-                });
-            }else{
-                const createdAccount= accounts.createAccount(req.body);
-                const user_token= {
-                    id: createdAccount.id,
-                    firstName: createdAccount.firstName,
-                    lastName: createdAccount.lastName,
-                    email: createdAccount.email,
-                    address: createdAccount.address,
-                    bio: createdAccount.bio,
-                    occupation: createdAccount.occupation,
-                    expertise: createdAccount.expertise,
-                    isAdmin: createdAccount.isAdmin,
-                    isAmentor: createdAccount.isAmentor
-                };
-
-                const token= jwt.sign(user_token, user_secret, { expiresIn: "1d" });
-                res.header('x-auth-token', token);
+            pool.connect((err, client, done) => {
+                const query = `SELECT * FROM users WHERE email = $1`; 
```

Comment:
Your controller is doing unnecessary work. Its main job should be to pass control to other methods and then return a response.

I would recommend that you create service files that will contain methods that interact with the database. You can place the service files in the database folder or create a new folder under `src` called services

Reply:
Ok. I am gonna work on it